gulpfile.js

```diff
@@ -140,10 +152,18 @@ gulp.task('prod', function (cb) {
     runSequence('build-prod', 'server', cb);
 });
 
-gulp.task('ci', ['build-ci'], function () {
-    return gulp.src(['dist/**', '!dist/hud.zip'])
+gulp.task('archive-dev', ['build-dev'], function () {
+    return gulp.src(['.dist/dev/**'])
+        .pipe(zip('hud.zip'))
```

Comment:
The intention was to have a hud.zip & hud-dev.zip in the dev folder (similiarly with client & server).  I was thinking this would be the least disruptive/confusing to any existing workflows that were expxecting to see hud.zip produced.  But maybe it's unecesary.  Or maybe we should do `hud-dev.zip` and `hud-prod.zip`, and eliminate `hud.zip` altogether.